packages/common/message-source.js

```diff
@@ -68,6 +68,7 @@ class StateMachineS3MessageSource extends MessageSource {
     this.originalMessage = message;
 
     if (message.resources) {
+      // BUCKET/KEY TO DO - what bucket should this be?
```

Comment:
Found a TODO, What's the status up this? And the line under?

Reply:
Good catch. So it turned out `resources` is something we do not use anymore so we don't need to change it. There is some legacy code hanging around that is used by other teams. I'll remove the to do.

Reply:
This is code that has been replaced by the message adapter btw